components/brave_federated/features.cc

```diff
@@ -12,23 +12,36 @@ namespace features {
 
 namespace {
 
+constexpr int kHoursInDay = 24;
+constexpr int kMinutesInHour = 60;
+constexpr int kSecondsInMinute = 60;
```

Comment:
see `base/time/time.h` for `kHoursPerDay` etc.

Reply:
perfect, was looking for that